stock_release_channel_geoengine/models/stock_release_channel.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import logging
+
+from odoo import fields, models
+
+_logger = logging.getLogger(__name__)
+
+
+class StockReleaseChannel(models.Model):
+
+    _inherit = "stock.release.channel"
+
+    restrict_to_delivery_zone = fields.Boolean()
+    delivery_zone = fields.GeoMultiPolygon()
```

Comment:
@sbejaoui To avoid problem with CI, can you import the GeoMultiPolygon from base_geoengine PLZ.